libs/application/templates/directorate-of-immigration/citizenship/src/forms/CitizenshipForm/ChildrenSupportingDocuments/ChildrenOtherDocumentsSubSection.ts

```
@@ -132,6 +170,23 @@ export const ChildrenOtherDocumentsSubSection = (index: number) =>
+            id: `${Routes.SUPPORTINGDOCUMENTS}.custodyDocumentsRequired`,
+            defaultValue: (application: Application) => {
+              const answers = application.answers as Citizenship
+              const selectedInCustody = getSelectedCustodyChildren(
+                application.externalData,
+                answers,
+              )
+              const thisChild =
+                !!selectedInCustody &&
+                selectedInCustody.find((_, i) => i === index - 1)
+              const hasOtherParent =
+                thisChild && !!thisChild.otherParent ? true : false
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)